src/tests/components/Post.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Post from '../../components/Post';
+import toJSON from 'enzyme-to-json';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+describe('Post Component', () => {
```

Comment:
This test should not be in this pull request. You will have to remove this commit from this branch. Look into:
- git reset head
- git force push